pages/login.js

```diff
@@ -1,5 +1,24 @@
 // TODO(alishaevn): redo this page when the time comes
+import { useSession, signIn, signOut } from 'next-auth/react'
 
-const Login = () => <h1>This is the login page.</h1>
+const Login = () => {
```

Comment:
Should we go ahead and remove this page before this is merged?

At the meeting we talked about having the login link direct to the auth flow directly. Otherwise we can leave it, and set that up when we look at the auth flow